Cookaholic_Backend/src/test/java/com/toolsofsoftwareprojects/cookaholic_backend/repo/UserRepoTests.java

```diff
@@ -0,0 +1,99 @@
+package com.toolsofsoftwareprojects.cookaholic_backend.repo;
+
+import com.toolsofsoftwareprojects.cookaholic_backend.model.CookaholicUser;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.annotation.Rollback;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@DataJpaTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class UserRepoTests {
+
+
+    @Autowired
+    private CookaholicUserRepo cookaholicUserRepo;
+
+
+    @Test
+    @Order(1)
+    @Rollback(value = false)
+    public void saveUser(){
+        CookaholicUser cookaholicUser = CookaholicUser.builder()
+                .email("daniel@a.com")
+                .userTitle("JavaScript")
+                .name("Daniel Craig")
+                .imageUrl("https://www.clipartmax.com/png/full/132-1327604_avatar-image-manager-color-icon-png.png")
+                .phone("06303456789")
+                .build();
+
+        cookaholicUser.setUserCode(UUID.randomUUID().toString());
+        cookaholicUserRepo.save(cookaholicUser);
+
+        Assertions.assertThat(cookaholicUser.getId()).isGreaterThan(0);
+    }
+    @Test
+    @Order(2)
+    public void getUsers(){
+        CookaholicUser cookaholicUser = cookaholicUserRepo.findById(1L).get();
+
+        Assertions.assertThat(cookaholicUser.getId()).isEqualTo(1L);
+    }
+
+    @Test
+    @Order(3)
+    public void getListOfUsersTest(){
+        List<CookaholicUser> cookaholicUsers = cookaholicUserRepo.findAll();
+        System.out.println("AAAAA"+cookaholicUsers.toString());
+
+        Assertions.assertThat(cookaholicUsers.size()).isGreaterThan(0);
+    }
+
+    @Test
+    @Order(4)
+    @Rollback(value = false)
+    public void updateUserTest(){
+
+        CookaholicUser cookaholicUser = cookaholicUserRepo.findById(1L).get();
+
+        cookaholicUser.setEmail("testing@gmail.com");
+
+        CookaholicUser employeeUpdated =  cookaholicUserRepo.save(cookaholicUser);
```

Comment:
Let's use cookaholicUserUpdated instead of employee.

Reply:
jogos májfren